app/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+// Use create-react-app defaults for JS and JSX.
+// See: https://www.gatsbyjs.org/docs/eslint/
+module.exports = {
+  globals: {
+    graphql: true,
+    __PATH_PREFIX__: true,
+  },
+  extends: `react-app`,
```

Comment:
Although our JavaScript guidelines recommend JSHint for linting, we can't use JSHint for linting Gatsby code, because [JSHint doesn't support JSX linting](https://stackoverflow.com/a/29756143). Here I used ESLint with a basic React config instead, which is the linter that Gatsby recommends (see the link in the comment).